src/core/storage/compression/Decompressor.h

```diff
@@ -256,7 +253,7 @@ class DecompressorImpl {
 				}
 				translate_file(new_path, size);
 			} else {
-				mkdir(new_path.c_str(), MkdirPermission);
+				fs::create_directory(new_path);
```

Comment:
This single-line change should not be placed in this commit. Please consider splitting this patch in two separate ones, where this change has a more descriptive summary, justifying its presence in this specific PR (e.g remove linux-specific dependencies).

Reply:
Done, check it out